csharp-intermediate-topics/RenameFilesWithCSharp/RenameFilesWithCSharp/FileFinderService.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace RenameFilesWithCSharp;
+
+public static class FileFinderService
+{
+    public static List<string> FindFilesInFolder(string directoryPath)
+    {
+        return Directory.EnumerateFiles(directoryPath, "*", SearchOption.AllDirectories)
+            .ToList();
+    }
```

Comment:
Is there a reason why we are returning a `List` here and using `ToList()` instead of just returning `IEnumerable` directly?